packages/frontend/src/pages/InferenceServers.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+import { Column, Row } from '/@/lib/table/table';
+import type { InferenceServer } from '@shared/src/models/IInference';
+import ServiceColumnName from '/@/lib/table/service/ServiceColumnName.svelte';
+import NavPage from '/@/lib/NavPage.svelte';
+import Table from '/@/lib/table/Table.svelte';
+import { inferenceServers } from '/@/stores/inferenceServers';
+import ServiceStatus from '/@/lib/table/service/ServiceStatus.svelte';
+import ServiceAction from '/@/lib/table/service/ServiceAction.svelte';
+
+const columns: Column<InferenceServer>[] = [
+  new Column<InferenceServer>('Status', { width: '50px', renderer: ServiceStatus, align: 'center' }),
+  new Column<InferenceServer>('Name', { width: '3fr', renderer: ServiceColumnName, align: 'center' }),
+  new Column<InferenceServer>('Action', { width: '50px', renderer: ServiceAction, align: 'center' }),
+];
+const row = new Row<InferenceServer>({});
+
+let data: InferenceServer[];
+$: data = $inferenceServers;
+</script>
+
+<NavPage title="Services" searchEnabled="{false}">
+  <svelte:fragment slot="content">
+    <div slot="content" class="flex flex-col min-w-full min-h-full">
```

Comment:
Shouldn't these have different slot names?

Reply:
That's a copy past from Models.svelte so I don't know

Reply:
Just verified and the inner div slot is never used.
So also on Models.svelte should be removed.
BTW as it does not give any issue it can be removed in another PR.